### PR TITLE
Check Access-Control-Allow-Private-Network earlier

### PR DESCRIPTION
**Pull request by jub0bs**
This fixes issue #86.

### CODE REVIEW

File: index.src.html [hunk 1]
```diff
@@ -727,8 +728,7 @@ <h4 id="cors-preflight">CORS preflight</h4>
                   to "`true`" in <var ignore>preflight</var>'s
                   [=request/header list=].
 
-      1.  Immediately after checking CORS-unsafe request-header names in
-          |response| against |request|'s [=request/header list=]:
+      1.  Immediately after the [=CORS check=]:
```
Comment (letitz) [inline index.src.html:731]:
```suggestion
      1.  Immediately after the [=CORS check=] succeeds:
```

**Review by letitz — CHANGES_REQUESTED** [review]
Thanks for sending this in! Please also rebuild `index.html` by running `make`.

File: index.src.html (hunk 1 above)
Comment (jub0bs) [reply index.src.html:731]: Done. FYI, I used the same version of bikeshed as you, as using the latest version of it introduced changes in index.html that were unrelated to my PR.

**Comment by jub0bs** [comment]
I'm noticing now that the generated `index.html` has more changes seemingly unrelated to my changes to `index.src.html` and I'm at a loss to explain them... Sorry, I'm not very familiar with bikeshed.

**Comment by letitz** [comment]
Hi there, sorry for the long wait - I was out for next to 2 months and left a bit earlier than anticipated.

Thanks for running `make`. It seems that something to do with `[=request/policy container=]` has changed since I last built `index.html`. I'll look into rebuilding and pushing an updated `index.html` over which you can rebase your PR cleanly.

**Comment by letitz** [comment]
Hmm, it seems that it had not been rebuilt after #72, but that does not explain the rest of the changes. Can you rebase and try again? If that does not work, I'll check out your PR, run `make`, and update the PR myself.

**Comment by jub0bs** [comment]
@letitz Sorry for not getting back to you. I somehow missed your reply.
I'll do as you suggested at some stage later today.

**Comment by letitz** [comment]
Thanks! Let me know how it goes.

**Comment by jub0bs** [comment]
That didn't work. I've still got what appears to be spurious changes.

FYI, I used bikeshed (3.11.4). I noticed that the latest `index.html` was generated with an older version of bikeshed. Perhaps its behaviour has changed in the interim. Not too sure...

**Comment by letitz** [comment]
Alright, thanks for trying! I'll amend the PR myself, and look into updating bikeshed :eyes:

**Comment by letitz** [comment]
Fixed, thanks for your contribution!

FWIW, I don't think your branch was rebased on top of the latest main, maybe that was the issue?

**Comment by jub0bs** [comment]
@letitz I thought I did rebase. Oh well, it's done now. Thanks!

By the way, is there a crbug associated to this change?

**Comment by letitz** [comment]
Maybe the issue was differing versions of bikeshed? Hopefully that's improved by 250f177cc5585fb18b4bfa0c20f7c07ebd79ce71

Otherwise, I just filed crbug.com/1424847.

**Pull request merged**